### DISCUSSION TITLE
Data annotations funciona em minial api?

### DISCUSSION THREAD

**Discussion by inacio88** [balta-io/forum#721] [Tira Dúvidas] [answered]
Estou seguindo o curso ainda e não sei se ainda irá voltar nesssa parte, mas criei aqui as classes de request normalmente tal qual está no curos, mas os testes que fiz aqui tentando ver se o DataAnnotations estava funcionando normalmente não deram certos: campos marcados com required mesmo mando sem eles no body é instanciando normalmente.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @inacio88, como vai? Nativamente as Minimal APIs não oferecem suporte aos DataAnnotations!
No curso vou mostrar como validar as entradas usando o Flunt!
